javascript/net/grpc/web/grpc_generator.cc

```diff
@@ -119,6 +124,34 @@ void PrintMessagesDeps(Printer* printer, const FileDescriptor* file) {
   printer->Print("\n\n\n");
 }
 
+void PrintCommonJsMessagesDeps(Printer* printer, const FileDescriptor* file) {
+  std::map<string, const Descriptor*> messages = GetAllMessages(file);
+  std::map<string, string> vars;
+  string package = file->package();
+  string filename = file->name();
+  // Remove .proto extension
+  filename = filename.substr(0, filename.size() - 6);
+  vars["package_name"] = package;
+  vars["filename"] = filename;
+
+  printer->Print(vars, "const proto = {};\n");
+  if(!package.empty()) {
```

Comment:
Can you add a `<space>` after each `if`, `while`, and `switch` for consistency please?